lessons/frameworks.ipynb

```diff
@@ -0,0 +1,1261 @@
+    "* In our example, the negative log likelihood:\n",
+    "\n",
+    "### $\\;\\;\\;\\;\\;\\chi^2 = \\boldsymbol{\\hat{\\theta}} - \\boldsymbol{\\theta})^T (\\mathcal{M}^T C^{-1} \\mathcal{M})^{-1} (\\boldsymbol{\\hat{\\theta}} - \\boldsymbol{\\theta}$, \n",
+    "\n",
+    "$\\;\\;\\;\\;\\;$where $C$ is the covariance matrix of the data (i.e. $C$ is diagonal, with elements equal to the squared uncertainties on each datapoint) and $\\mathcal{M}$ is the 2xN _design matrix_ that predicts data given parameters via   $\\mathcal{M}\\boldsymbol{\\theta} = \\boldsymbol{m}$.\n",
+    "\n",
+    "* This $\\chi^2$ function can be computed on a grid, and visualized as a contour plot: the contour at $\\chi^2_{\\rm min} + 2.30$ will enclose the 68% 2-D confidence region."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Approximate uncertainties\n",
+    "\n",
+    "* In general, the covariance matrix of a _Gaussian approximation to the likelihood_ can be calculated by taking second derivatives of the log likelihood at the peak, and inverting the resulting _Hessian_ matrix. \n",
+    "\n",
+    "* This gives a lower limit to the covariance of a set of estimators: \n",
+    "\n",
+    "## $\\;\\;\\;\\;\\;V^{-1}_{ij} \\geq -\\frac{\\partial^2 \\log{L}}{\\partial\\theta_i\\partial\\theta_j} \\biggr|_{\\boldsymbol{\\hat{\\theta}}}$\n",
+    "\n",
+    "> $V$ is what [`scipy.optimize.minimize`](https://docs.scipy.org/doc/scipy-0.18.1/reference/generated/scipy.optimize.minimize.html) returns in its `hess_inv` field if you pass it the negative log likelihood."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Wording\n",
+    "\n",
+    "* Frequentist confidence intervals are different from Bayesian credible regions:\n",
+    "\n",
+    "> \"68% of datasets would give a 68% frequentist confidence interval that contains the true parameter value\"\n",
+    "\n",
+    "> \"The probability of the true parameter value lying within the 68% Bayesian credible region is 68%\"\n",
+    "\n",
+    "* The difference in wording comes from the two different definitions of probability"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Uncertainties in the estimators\n",
+    "\n",
+    "* The covariance matrix of a Gaussian approximation to the likelihood defines a 1-sigma, 2D, elliptical, _frequentist confidence region_\n",
+    "\n",
+    "* Since this came from transforming the sampling distribution, which is a PDF over datasets, the confidence interval enables conclusions in terms of fractions of an ensemble of (hypothetical) datasets\n",
+    "\n",
+    "* The 68% confidence interval is the region that we expect to contain the true parameter value 68% of the time"
```

Comment:
Maybe it's been said enough times, but emphasize that "X% of the time" implies a fictional universe in which we can repeat an experiment independently as many times as we want to.

Reply:
Done!